solutions/security/detect-and-alert/suppress-detection-alerts.md

```diff
@@ -13,74 +13,60 @@ products:
-* [New terms](/solutions/security/detect-and-alert/create-detection-rule.md#create-new-terms-rule)
-* [{{esql}}](/solutions/security/detect-and-alert/create-detection-rule.md#create-esql-rule)
-* [{{ml-cap}}](/solutions/security/detect-and-alert/create-detection-rule.md#create-ml-rule)
-
-Normally, when a rule meets its criteria repeatedly, it creates multiple alerts, one for each time the rule’s criteria are met. When alert suppression is configured, duplicate qualifying events are grouped, and only one alert is created for each group. Depending on the rule type, you can configure alert suppression to create alerts each time the rule runs, or once within a specified time window. You can also specify multiple fields to group events by unique combinations of values.
+Depending on the rule type, you can configure alert suppression to create alerts each time the rule runs, or once within a specified time window. You can also specify multiple fields to group events by unique combinations of values.
 
 The {{security-app}} displays several indicators in the Alerts table and the alert details flyout when a detection alert is created with alert suppression enabled. You can view the original events associated with suppressed alerts by investigating the alert in Timeline.
 
 ::::{note}
-Alert suppression is not available for Elastic prebuilt rules. However, if you want to suppress alerts for a prebuilt rule, you can duplicate it, then configure alert suppression on the duplicated rule.
+
+When modifying Elastic prebuilt rules, you can also add on alert suppression. Refer to [Modify existing rules settings](/solutions/security/detect-and-alert/manage-detection-rules.md#edit-rules-settings) to learn about the requirements for editing prebuilt rules and the rule settings that you change.
+
 ::::
 
 
 ## Configure alert suppression [security-alert-suppression-configure-alert-suppression]
 
-You can configure alert suppression when you create or edit a supported rule type. Refer to documentation for creating [custom query](/solutions/security/detect-and-alert/create-detection-rule.md#create-custom-rule), [threshold](/solutions/security/detect-and-alert/create-detection-rule.md#create-threshold-rule), [event correlation](/solutions/security/detect-and-alert/create-detection-rule.md#create-eql-rule), [new terms](/solutions/security/detect-and-alert/create-detection-rule.md#create-new-terms-rule), [{{esql}}](/solutions/security/detect-and-alert/create-detection-rule.md#create-esql-rule), or [{{ml}}](/solutions/security/detect-and-alert/create-detection-rule.md#create-ml-rule) rules for detailed instructions.
+::::{admonition} Requirements and notices
+* In {{stack}} alert suppression requires a [Platinum or higher subscription](https://www.elastic.co/pricing). In [{{serverless-short}}, you must have the appropriate [{{serverless-short}} project tier](../../../deploy-manage/deploy/elastic-cloud/project-settings.md).
```

Comment:
```suggestion
* In {{stack}} alert suppression requires a [Platinum or higher subscription](https://www.elastic.co/pricing). In {{serverless-short}}, you must have the appropriate [{{serverless-short}} project tier](../../../deploy-manage/deploy/elastic-cloud/project-settings.md).
```

Reply:
If I'm remembering correctly, we might not want to say specific subscription levels in the docs, because those things could change and we can keep the pricing website as the single source of truth. That being said, I don't see alert suppression on the pricing page. I could also be remembering incorrectly. @bmorelli25 can probably shine some light here.

Reply:
Also, I verified that alert suppression is mentioned on the pricing pages for [cloud hosted](https://www.elastic.co/pricing/cloud-hosted?cta=sec) and [self-hosted](https://www.elastic.co/subscriptions), but couldn't find it on the serverless pricing [page](https://www.elastic.co/pricing/serverless-security) for Security. @approksiu should the suppression feature be highlighted there?

Reply:
LGTM!